plugins/producer_plugin/include/eosio/producer_plugin/subjective_billing.hpp

```diff
@@ -171,10 +171,12 @@ class subjective_billing {
       _block_subjective_bill_cache.clear();
    }
 
-   void on_block( const block_state_ptr& bsp, const fc::time_point& now ) {
-      if( bsp == nullptr ) return;
+   void on_block( fc::logger& log, const block_state_ptr& bsp, const fc::time_point& now ) {
+      if( bsp == nullptr || _disabled ) return;
```

Comment:
Is there any way for this to cause a change in behavior?  It assumes that if `_disabled` is true, then there is nothing to remove, which should be valid as long as `_disabled` does not change.

Reply:
`_disabled` is private and only can be turned to `true` via `disabled()`